### PR TITLE
Fixes #507: Show toast before auto filling credentials

### LINKED ISSUE #507: Credentials should not be autofilled by default
### Description: 
Saved credentials are auto-populated.

### Steps to Reproduce 
  1. Navigate to github.com
  2. Login to github.com and save credentials 
  3. Logout and navigate to github.com/login
  4. Credentials will be populated by default.

**Actual result:** 
Credentials are populated by default.

**Expected result:**
Require user interaction to populate credentials

**Reproduces how often:** [Easily reproduced, Intermittent Issue]
Always

**Brave Version:** 
Development (Commit - 60a82d37c7987162971f1d23bf5a8639131a27b5)

**Device details:** 
Simulator (iPhone XS - iOS 12.1)

### PR DESCRIPTION
**Pull request by AasthaGupta**
## Summary of Changes

This pull request fixes #507 

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Comment by kylehickinson** [comment]
Hi @AasthaGupta, first of all thanks for the PR! :) 

I just tested out the code though and it seems like it will show the toast regardless of if you have credentials for that website (tested it on https://gitbub.com/login) so it seems like this may need a bit more work to get the correct behavior (Probably need code to check that the credential exists before showing the toast is all)

Also we may want to include an option for users to do this, but I think we'll have to talk to product to figure that out so I wouldn't worry about it for now :)

**Comment by AasthaGupta** [comment]
@kylehickinson Sorry about missing it out. I have added the check now.

File: Client/Frontend/Browser/LoginsHelper.swift [hunk 1]
```diff
@@ -210,8 +210,25 @@ class LoginsHelper: TabContentScript {
 
             let json = JSON(jsonObj)
             let src = "window.__firefox__.logins.inject(\(json.stringValue()!))"
-            self.tab?.webView?.evaluateJavaScript(src, completionHandler: { (obj, err) -> Void in
+
+            guard !json["logins"].isEmpty else {
+                self.tab?.webView?.evaluateJavaScript(src, completionHandler: { (obj, err) -> Void in
+                })
```
Comment (kylehickinson) [inline Client/Frontend/Browser/LoginsHelper.swift:216]: Sorry for this being so late 🙂 Is there a reason you evaluate the JS here if logins isn't empty? Shouldn't it just return?

**Review by kylehickinson — CHANGES_REQUESTED** [review]

**Comment by iccub** [comment]
We are moving to chromium's autofill implementation soon, going to close this ticket for now

**Pull request closed without merging**